docs/source/queue_submit_flow.md

```diff
@@ -0,0 +1,80 @@
+# How pyiron submits a job into the queue
+
+If you just want to configure the queue setup, look into the [documentation](https://pyiron.readthedocs.io/en/latest/source/installation.html#hpc-cluster-with-postgresql-database-and-jupyterhub). The following details on the code flow for job submission to the queue.
+
+Every time pyiron submits a job to the queue (reachable from the current location - for remote setup this is run on the remote machine) it runs:
+
+https://github.com/pyiron/pyiron_base/blob/b1e188458e96ae2fe71591ffef2769748481c204/pyiron_base/jobs/job/runfunction.py#L406-L420
+
+The job submission is handled by the queue adapter which populates the slurm run template 
+
+```bash
+#!/bin/bash
+{%- if cores < 40 %}
+#SBATCH --partition=s.cmfe
+{%- else %}
+#SBATCH --partition=p.cmfe
+{%- endif %}
+#SBATCH --ntasks={{cores}}
+#SBATCH --constraint='[swi1|swi1|swi2|swi3|swi4|swi5|swi6|swi7|swi8|swi9]'
+{%- if run_time_max %}
+#SBATCH --time={{ [1, run_time_max // 60]|max }}
+{%- endif %}
+{%- if memory_max %}
+#SBATCH --mem={{memory_max}}G
+{%- else %}
+{%- if cores < 40 %}
+#SBATCH --mem-per-cpu=3GB
+{%- endif %}
+{%- endif %}
+#SBATCH --output=time.out
+#SBATCH --error=error.out
+#SBATCH --job-name={{job_name}}
+#SBATCH --chdir={{working_directory}}
+#SBATCH --get-user-env=L
+
+pwd;
+echo Hostname: `hostname`
+echo Date: `date`
+echo JobID: $SLURM_JOB_ID
+
+{{command}}
+```
+(see https://github.com/eisenforschung/pyiron-resources-mpie/blob/master/queues/cmti.sh)
```

Comment:
This is only accessible for MPIE - better link to https://github.com/pyiron/pysqa/tree/master/tests/config/slurm/slurm.sh